### PR TITLE
Fix/yarn lock

### PR DESCRIPTION
**Pull request by marie-j**

### CODE REVIEW

**Comment by antleblanc** [comment]
Based on the following commit hash:

6b46266671a342ef4cd038d07ad9984dab828788 (latest commit on `develop` branch)

Install dependencies:

```sh
$ yarn install
```

Expected diff:

```diff
diff --git a/yarn.lock b/yarn.lock
index be3191e72b0d..5b4c797e4d42 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -3236,6 +3236,11 @@ ansi-regex@^2.0.0:
   resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-2.1.1.tgz#c3b33ab5ee360d86e0e628f0468ae7ef27d654df"
   integrity sha1-w7M6te42DYbg5ijwRorn7yfWVN8=
 
+ansi-regex@^3.0.0:
+  version "3.0.0"
+  resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-3.0.0.tgz#ed0317c322064f79466c02966bddb605ab37d998"
+  integrity sha1-7QMXwyIGT3lGbAKWa922Bas32Zg=
+
 ansi-regex@^4.1.0:
   version "4.1.0"
   resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-4.1.0.tgz#8b9f8f08cf1acb843756a839ca8c7e3168c51997"
```

Environment tested:

```sh
$ node --version
v12.16.1
$ yarn --version
1.22.0
```

File: yarn.lock [hunk 1]
```diff
@@ -16276,8 +16281,6 @@ strip-ansi@^4.0.0:
   version "4.0.0"
   resolved "https://registry.yarnpkg.com/strip-ansi/-/strip-ansi-4.0.0.tgz#a8479022eb1ac368a871389b635262c505ee368f"
   integrity sha1-qEeQIusaw2iocTibY1JixQXuNo8=
-  dependencies:
```
Comment (antleblanc) [inline yarn.lock]: Still defined as a `dependency` for `strip-ansi`:

see: https://github.com/chalk/strip-ansi/blob/v4.0.0/package.json#L46

Comment (FredericEspiau) [reply yarn.lock]: I don't get why `yarn` keeps on removing and adding it from PR to PR 🤔

**Comment by marie-j** [comment]
> Based on the following commit hash:
> 
> [6b46266](https://github.com/ovh/manager/commit/6b46266671a342ef4cd038d07ad9984dab828788) (latest commit on `develop` branch)
> 
> Install dependencies:
> 
> ```shell
> $ yarn install
> ```
> 
> Expected diff:
> 
> ```diff
> diff --git a/yarn.lock b/yarn.lock
> index be3191e72b0d..5b4c797e4d42 100644
> --- a/yarn.lock
> +++ b/yarn.lock
> @@ -3236,6 +3236,11 @@ ansi-regex@^2.0.0:
>    resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-2.1.1.tgz#c3b33ab5ee360d86e0e628f0468ae7ef27d654df"
>    integrity sha1-w7M6te42DYbg5ijwRorn7yfWVN8=
>  
> +ansi-regex@^3.0.0:
> +  version "3.0.0"
> +  resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-3.0.0.tgz#ed0317c322064f79466c02966bddb605ab37d998"
> +  integrity sha1-7QMXwyIGT3lGbAKWa922Bas32Zg=
> +
>  ansi-regex@^4.1.0:
>    version "4.1.0"
>    resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-4.1.0.tgz#8b9f8f08cf1acb843756a839ca8c7e3168c51997"
> ```
> 
> Environment tested:
> 
> ```shell
> $ node --version
> v12.16.1
> $ yarn --version
> 1.22.0
> ```

@antleblanc 
```shell
$ node --version
v12.16.0
$ yarn --version
1.22.0
```

File: yarn.lock (hunk 1 above)
Comment (antleblanc) [reply yarn.lock]: Probably related to multiple versions of Yarn. I will merge it and keep a closer look.

**Pull request merged**